modules/appeals_api/spec/services/appeals_api/pdf_construction/generator_spec.rb

```diff
@@ -41,6 +41,31 @@
           Timecop.return
         end
       end
+
+      context 'extra long preferred email verification' do
+        let(:notice_of_disagreement) { create(:notice_of_disagreement) }
+        let(:new_email) { Faker::Lorem.characters(number: 255 - 6) + '@a.com' }
+
+        before do
+          notice_of_disagreement.form_data['data']['attributes']['veteran']['emailAddressText'] = new_email
+          notice_of_disagreement.save!
+        end
+
+        it 'places long email addresses onto additional page' do
+          Timecop.freeze(Time.zone.parse('2020-01-01T08:00:00Z'))
+          generated_pdf = described_class.new(notice_of_disagreement).generate
+          reader = PDF::Reader.new(generated_pdf)
+          # Character representation of pdf can be hard to test against, since its layout is converted to line breaks,
+          # whitespace, & other special characters in non-intuitive ways. We do our best to work with it, below.
```

Comment:
What do you think about including a max length email in our `expected_10182_extra.pdf` test? Then we wouldn't have to do this character by character testing like this?

Reply:
I keep hoping that we can do tests that don't need a rendered fixture file to check against - mostly because it _feels_ like I'm actually testing a specific piece of functionality instead of a mashup of "all things that may produce an extra page."

That said, I agree with hindsight that the fixture file is probably the better way to make this less piecemeal to work around `pdf -> text` craziness. I'll get that updated tomorrow :)